client/packages/system/src/Patient/PatientView/AppBarButtons.tsx

```diff
@@ -28,6 +32,7 @@ export const AppBarButtons: FC<{ disabled: boolean }> = ({ disabled }) => {
     },
   });
   const disableEncounterButton = enrolmentData?.nodes?.length === 0;
+  if (!store?.preferences.omProgramModule) return null;
```

Comment:
I would expect the `Print` button to still be available for non-program patients - shouldn't it?